packages/create/templates/dbos-drizzle/src/main.ts

```diff
@@ -91,7 +91,7 @@ app.get('/greeting/:name', (req: Request, res: Response) => {
 async function main() {
   DBOS.setConfig({
     name: 'dbos-drizzle',
-    databaseUrl: process.env.DBOS_DATABASE_URL,
+    systemDatabaseUrl: process.env.DBOS_SYSTEM_DATABASE_URL,
```

Comment:
we should be adding `dotenv` package to all these template apps

Reply:
Looks like only the Prisma app requires modifying the env variables.